bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/api/AgsMagnitude.scala

```diff
@@ -23,7 +24,7 @@ object AgsMagnitude {
    * Gets the widest possible range limits incorporating the given conditions
    * and speeds.
    */
-  def rangeLimits(mc: MagnitudeCalc, c1: (Conditions, GuideSpeed), c2: (Conditions, GuideSpeed)): MagnitudeRange =
+  private def rangeConstraints(mc: MagnitudeCalc, c1: (Conditions, GuideSpeed), c2: (Conditions, GuideSpeed)): Option[MagnitudeConstraints] =
```

Comment:
Turning this into an `Option` seems like it would have ripple effects.  What changed to make this optional now?